packages/nc-gui-v2/composables/useTheme/index.ts

```diff
@@ -12,36 +12,81 @@ const [setup, use] = useInjectionState((config?: Partial<ThemeConfig>) => {
   const primaryColor = useCssVar('--color-primary', typeof document !== 'undefined' ? document.documentElement : null)
   const accentColor = useCssVar('--color-accent', typeof document !== 'undefined' ? document.documentElement : null)
 
-  /** current theme config */
-  const currentTheme = useStorage<ThemeConfig>(
-    `${NOCO}db-theme`,
-    {
+  const { project, getProjectMeta, updateProject } = useProject()
+
+  const route = useRoute()
+
+  // set theme based on active project
+  watch(project, () => {
+    setTheme({
       primaryColor: themeV2Colors['royal-blue'].DEFAULT,
       accentColor: themeV2Colors.pink['500'],
+      ...getProjectMeta()?.theme,
+    })
+  })
```

Comment:
Please don't make `useTheme` dependant on the project itself.
Instead feed the project theme into the composable as an argument.
I.e. move this code into the page instead of this composable.